react-basics/react-basics_attributes/src/App.jsx

```diff
@@ -1,5 +1,14 @@
 import "./App.css";
 
 export default function App() {
-  return <div>Replace me with an article</div>;
+  return <Article/>;
+}
+
+export function Article () {
+  return <article className="article">
+  <h2 className="article__title">Text of my Choice</h2>
+  <label htmlFor="input__field">Some input</label>
+  <input id="input__field"></input>
+  <a className="article__link" href="https://en.wikipedia.org/wiki/React_(JavaScript_library)"></a>
```

Comment:
Something is missing in the `<a>` tag. Were you able to see the link in the website?